shimmingtoolbox/cli/check_env.py

```diff
@@ -149,6 +175,23 @@ def get_dcm2niix_version() -> str:
     return version_output
 
 
+def get_sct_version() -> str:
```

Comment:
Just a comment: All of these SCT things should be pulled into a DDD domain. I believe I've already done this in my PR. but I'll check that on my end. check_env needs to call domains that have the logic for each of these that need to be checked, not just have a long file because it'll be easier to read/reuse